parm/soca/mom6/diag_table

```diff
@@ -0,0 +1,238 @@
+20210322.06Z.C48.64bit.non-mono
+2021 03 22 06 0 0
+
+"fv3_history",    0,  "hours",  1,  "hours",  "time"
+"fv3_history2d",  0,  "hours",  1,  "hours",  "time"
+######################
+"ocn_da%4yr%2mo%2dy%2hr",      0,  "hours", 1, "hours", "time", 1, "hours", "1901 1 1 0 0 0"
+##############################################
+# static fields
+  "ocean_model", "geolon",      "geolon",      "ocn_da%4yr%2mo%2dy%2hr", "all", .false., "none", 2
+  "ocean_model", "geolat",      "geolat",      "ocn_da%4yr%2mo%2dy%2hr", "all", .false., "none", 2
+# ocean output TSUV and others
+ "ocean_model", "SSH",       "SSH",      "ocn_da%4yr%2mo%2dy%2hr","all",.true.,"none",2
+#"ocean_model", "SST",       "SST",      "ocn_da%4yr%2mo%2dy%2hr","all",.false.,"none",2
+#"ocean_model", "SSS",       "SSS",      "ocn_da%4yr%2mo%2dy%2hr","all",.false.,"none",2
+ "ocean_model", "MLD_0125",  "MLD",      "ocn_da%4yr%2mo%2dy%2hr","all",.false.,"none",2
+#===============================================
+
+
+# Z-Space Fields Provided for CMIP6 (CMOR Names):
```

Comment:
Comment doesn't match why we're saving these fileds. Remove or update the comment.